content/CORE/UIReference/TopMenu/UIPreferences.md

```diff
@@ -12,80 +16,82 @@ aliases: /core/administration/uipreferences/
+| **theme name (variable)** | checkbox | Each custom theme is listed. |
+| **DELETE SELECTED** | button | Removes each selected custom theme from the system. |
+| **CREATE NEW THEME** | button | Opens the theme creation screen. |
 
 ## Custom Theme Editor
 
-![Preferences Custom Theme](/images/CORE/12.0/PreferencesCustomTheme.png "Custom UI Theme")
+![Preferences Custom Theme](/images/CORE/13.0/PreferencesCustomTheme.png "Custom UI Theme")
 
 ### Create Theme
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| Load colors from existing theme | drop down | Imports the settings from the chosen theme into the **Create Theme** and **Preview** tabs. |
-| GENERAL | tab | Contains the primary options for the new theme. |
-| COLORS | tab | Contains options for setting custom color values. |
-| PREVIEW | tab | Controls the **Preview** widget. |
+| **Load colors from existing theme** | dropdown list | Imports the settings from the chosen theme into the **Create Theme** and **Preview** tabs. |
+| **GENERAL** | tab | Contains the primary options for the new theme. |
+| **COLORS** | tab | Contains options for setting custom color values. |
+| **PREVIEW** | tab | Controls the **Preview** widget. |
 
 #### GENERAL
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| Custom Theme Name | text input | Enter a name to identify the new theme. |
-| Menu Label | text input | Enter a short name to use in the TrueNAS web interface menus |
-| Description | text input | Enter a short description of the new theme. |
-| Choose Primary | drop down | Choose from either a generic color or import a specific color setting to use as the primary theme color. |
-| Choose Accent | drop down | Choose from either a generic color or import a specific color setting to use as the accent color for the theme. |
-| Choose Topbar | drop down | Changes the color of the top menu bar in the web interface. |
-| SUBMIT | button | Save the current selections and create the new theme. |
-| CANCEL | button | Return to the **Preferences** screen without creating a new theme. |
+| **Custom Theme Name** | text input | Enter a name to identify the new theme. |
+| **Menu Label** | text input | Enter a short name to use in the TrueNAS web interface menus. |
+| **Description** | text input | Enter a short description of the new theme. |
+| **Choose Primary** | dropdown list | Choose from either a generic color or import a specific color setting to use as the primary theme color. |
+| **Choose Accent** | dropdown list | Choose from either a generic color or import a specific color setting to use as the accent color for the theme. |
+| **Choose Topbar** | dropdown list | Changes the color of the top menu bar in the web interface. |
+| **SUBMIT** | button | Save the current selections and create the new theme. |
+| **CANCEL** | button | Return to the **Preferences** screen without creating a new theme. |
 
```

Comment:
Same comment as above, to remove the Style column and incorporate into the description. For a text input, a description can be something like this, " Enter a name to identify the new theme." 
Standardizing on select not choose for dropdown lists, we use choose when the customer is going to create a name for their setting or host name or whatever they decide on. Select is when we provide the list of options.